sources/packages/web/src/components/institutions/request-a-change/InprogressOfferingChangeSummary.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <tab-container>
+    <body-header-container>
+      <template #header>
+        <body-header title="Applications" :recordsCount="applications?.count">
+          <template #subtitle>
+            Waiting for the student and StudentAid BC decision on the requested
+            change
+          </template>
+          <template #actions>
+            <v-text-field
+              density="compact"
+              label="Search name or application #"
+              variant="outlined"
+              v-model="searchCriteria"
+              @keyup.enter="searchApplicationOfferingChangeRecords"
+              prepend-inner-icon="mdi-magnify"
+              hide-details="auto"
+            >
+            </v-text-field>
+          </template>
+        </body-header>
```

Comment:
The template header of the body header container is suppose to have only `<body-header>` right? the content-group and data-table must be in the default slot.

Reply:
nice catch